source/User_Guide/Settings/subusers.md

```diff
@@ -0,0 +1,199 @@
+  * Password - The password your Subuser will use to access SendGrid.com.
+  * Avatar - A [gravatar](http://gravatar.com) will load when an email address is entered, making it easier to identify your Subusers in the Subuser list.
+
+1. Select the IP Address you wish to associate with the Subuser account. 
+   When you create a SendGrid account and select a Pro 100k plan or higher, your account is assigned to a dedicated IP address. All of your mail sent through SendGrid will show as originating from this IP address, unless you have purchased additional IPs and assigned IP addresses to them or set up [IP Pools]({{root_url}}/API_Reference/Web_API_v3/IP_Management/ip_pools.html). You can see which other users are assigned to the same IP address, so you can keep similar senders together.
+   
+If you need more IP addresses, please contact [SendGrid support]({{site.support_url}}).
+
+{% anchor h2 %}
+Manage Subusers
+{% endanchor %}
+
+Subusers can be managed via [Subuser API](https://sendgrid.api-docs.io/v3.0/Subusers-api) or by using the SendGrid UI.
+
+*To manage a Subuser from the SendGrid UI:*
+
+1. Navigate to **Settings** and select **Subuser Management**.
+1. Select the name of the Subuser. This takes you to the Subuser’s account and profile settings. From there, you can manage and update your Subuser Settings.
+
+{% anchor h2 %}
+Optional Profile Information
+{% endanchor %}
+
+We do not require this information for your Subusers, but we suggest that you fill out as much as possible to improve the deliverability for this Subuser, especially if the information does not match your parent account’s information. This will also allow SendGrid to contact your Subuser directly if we notice anything on their account.
+
+**First Name** - The first name of the person responsible for this Subuser.
+
+**Last Name** - The last name of the person responsible for this Subuser.
+
+**Company** - The name of your Subuser’s company.
+
+**Address 1** - The first line of your Subuser’s address.
+
+**Address 2** - The second line of your Subuser’s address.
+
+**City** - The city in which your Subuser operates.
+
+**State** - The state or province in which your Subuser operates.
+
+**Country** - The country in which your Subuser operates.
+
+**Zip** - The zip code in which your Subuser operates.
+
+**Company Phone Number** - The phone number for your Subuser’s business.
+
+**Company Website** - The website for your Subuser’s business.
+
+{% anchor h2 %}
+Allocating or removing credits from a Subuser
+{% endanchor %}
+
+{% warning %}
+This feature only makes changes to a Subuser when the nonrecurring credit option is selected for this Subuser.
+{% endwarning %}
+
+If your Subuser has been allocated nonrecurring credits, then this feature will allow you to allocate credits to, or remove credits from this Subuser account. Simply click **Change Credit Rules** and then choose whether to add or subtract credits for this Subuser.
```

Comment:
To allocate Subuser credits, click **Change Credit Rules** and then choose whether to add or subtract credits.

What page is this on? Billing in the parent account?